CMakeLists.txt

```diff
@@ -0,0 +1,220 @@
+# Advanced settings
+option(KokkosTools_REUSE_KOKKOS_COMPILER "Set the compiler and flags based on installed Kokkos settings" OFF)
+mark_as_advanced(KokkosTools_REUSE_KOKKOS_COMPILER)
+
+# Fetch Kokkos options:
+acquire_kokkos_config()
+if(DEFINED Kokkos_FOUND_MSG)
+  message(STATUS "${Kokkos_FOUND_MSG}: ${Kokkos_INSTALL_DIR}\n"
+    "\t\tDevices: ${Kokkos_DEVICES}\n"
+    "\t\tArchitecture: ${Kokkos_ARCH}\n"
+    "\t\tTPLs: ${Kokkos_TPLS}\n"
+    "\t\tCompiler: ${Kokkos_CXX_COMPILER} (${Kokkos_CXX_COMPILER_ID})\n"
+    "\t\tCMAKE_CXX_FLAGS: ${CMAKE_CXX_FLAGS}\n"
+    "\t\tOptions: ${Kokkos_OPTIONS}")
+  # Synchronize compiler and flags (only when explicitly requested)
+  if(KokkosTools_REUSE_KOKKOS_COMPILER)
+    set(CMAKE_CXX_COMPILER "${Kokkos_CXX_COMPILER}"        CACHE STRING "C++ Compiler")
+    set(CMAKE_CXX_STANDARD "${CMAKE_CXX_STANDARD_DEFAULT}" CACHE STRING "C++ Standard: 98, 11, 14, 17, 20 or 23")
+  endif()
+else()
+  if(KokkosTools_REUSE_KOKKOS_COMPILER)
+    message(FATAL_ERROR "Kokkos not found: can't reuse Kokkos compiler (which was explicitly"
+                        "requested with KokkosTools_REUSE_KOKKOS_COMPILER=ON)")
+  endif()
+  message(STATUS "Kokkos NOT found")
+endif()
+
+# Libraries
+if(KokkosTools_ENABLE_PAPI)
+  find_package(PAPI REQUIRED) # TODO: papi-connector requires v6.0 or newer
+  cmake_path(GET PAPI_INCLUDE_DIR PARENT_PATH PAPI_ROOT)
+  message(STATUS "Found PAPI ${PAPI_VERSION_STRING} at ${PAPI_ROOT}")
+  set(KokkosTools_HAS_PAPI ON)
+else()
+  message(STATUS "PAPI support disabled")
+  set(KokkosTools_HAS_PAPI OFF)
+endif()
+
+if(KokkosTools_ENABLE_MPI)
+  find_package(MPI REQUIRED)
+  message(STATUS "Found MPI ${MPI_CXX_VERSION}: ${MPI_CXX_LIBRARIES}")
+  set(USE_MPI 1)
+else()
+  set(USE_MPI 0)
+endif()
+
+include(cmake/configure_variorum.cmake)
+
+set(KOKKOSTOOLS_HAS_CALIPER ${KokkosTools_ENABLE_CALIPER})
+set(KOKKOSTOOLS_HAS_NVPROF ${Kokkos_ENABLE_CUDA})
+
+if(DEFINED ENV{VTUNE_HOME})
+  set(VTune_ROOT $ENV{VTUNE_HOME})
+endif()
+if(VTune_ROOT)
+  find_package(ITT REQUIRED)
+  set(KOKKOSTOOLS_HAS_VTUNE ON)
+else()
+  message(WARNING "Set VTUNE_HOME in environment or VTune_ROOT in build options to build VTune connectors")
+  set(VTune_ROOT "" CACHE STRING "Path to VTune Intel compiler")
+  set(KOKKOSTOOLS_HAS_VTUNE OFF)
+endif()
+
+# make Kokkos profiling interface available for native profilers
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/profiling/all)
+
+# Config file
+configure_file(common/kp_config.hpp.in common/kp_config.hpp)
+set(COMMON_HEADERS_PATH ${CMAKE_CURRENT_BINARY_DIR}/common)
+include_directories(${COMMON_HEADERS_PATH})
+
+set(SINGLELIB_PROFILERS "" CACHE STRING "" FORCE)
+
+# Export settings
+include(GNUInstallDirs)
+set(EXPORT_NAME KokkosToolsConfig)
+set(EXPORT_INCLUDE_DIR ${CMAKE_INSTALL_INCLUDEDIR})
+set(EXPORT_LIB_DIR ${CMAKE_INSTALL_LIBDIR})
+set(EXPORT_TARGETS "" CACHE STRING "" FORCE)
+
+if(WIN32 OR OSX)
```

Comment:
OSX is not a CMake variable and the message below would be wrong anyway.

Reply:
This is now fixed in the cmake-build branch of  github.com/NextGenAnalytics/kokkos-tools FYI.